.devcontainer/README.md

```diff
@@ -12,37 +12,47 @@ All configuration related to the development container is in the `.devcontainer`
+  > **NOTE:** When VS Code starts, it reads the file `.vscode/extensions.json` relative from the current working directory. On startup, VS Code may prompt the user to install any extensions referenced here that are not already installed.
 
 ### Step-by-Step
 
-1. Open VS Code to the root folder of the git repository (not a sub folder or a parent folder). You should see a folder named `.devcontainer` at the root of the VS Code Explorer pane.
-
-    > NOTE: If you are using WSL or BASH on Linux or Mac, you can navigate to the root folder of the git repository and enter the command below to launch VS Code and open it to the right directory. Be sure to include the trailing ".".
-
-    ```BASH
-    code .
-    ```
+1. Open VS Code from the root folder of the local workspace (not a sub folder or a parent folder). You should see a folder named `.devcontainer` at the root of the VS Code Explorer pane.
 
 1. In the VS Code command palette (Ctrl+Shift+P), run this command
 
     ```VSCODE
     Remote-Containers: Reopen in Container
     ```
 
-    The container will build on your machine. The first build may take some time; the `Reopen in Container` command will be much faster after the initial container build, and VS Code will prompt you if the container needs to be rebuilt when the `Dockerfile` or container configuration settings have changed.
+    The container will build on your machine. The first build may take several minutes; the `Reopen in Container` command will be much faster after the initial container build, and VS Code will prompt you if the container needs to be rebuilt when the `Dockerfile` or container configuration settings have changed.
 
-1. (Optional) if you'd like to interact with the devcontainer's terminal from another terminal other than VS Code's built in terminal, you can docker's `exec` command.
+1. (*Optional*) If you'd like to interact with the devcontainer's terminal from another terminal other than VS Code's built in terminal, you can use the `docker exec` command.
 
-    > NOTE: VS Code attaches to the container as the user named "vscode", so you have to do the same thing when attaching to a BASH session in the container by specifying the user as an argument to the `docker exec` command. If you do not specify the user then you will be connected as root, which will cause permissions issues in git (if you are launching VS Code from WSL).
+    > **NOTE:** VS Code attaches to the container as the user named "vscode", so you have to do the same thing when attaching to a BASH session in the container by specifying the user as an argument to the `docker exec` command. If you do not specify the user then you will be connected as root, which will cause permissions issues in git (if you are launching VS Code from WSL).
 
     ```BASH
     docker exec --interactive --tty --user vscode missionlz-dev /bin/bash
```

Comment:
Please fix the last sentence in this file because it is not complete. (This is not related to a change you made.) It should read something like: When you are logged into the dev container's terminal, you will find the working director at `/workspaces/missionlz$`

Reply:
I moved this up to the prior bullet (e.g. with the step to "Remote-Containers: Reopen in Container") because it occurs with this step